docs/tutorials/podman_tutorial.md

```diff
@@ -96,6 +96,28 @@ After being restored, the container will answer requests again as it did before
 curl http://<IP_address>:8080
 ```
 
+### Migrate the container
+To live migrate a container from one host to another the container is checkpointed on the source
+system of the migration, transfered to the destination system and then restored on the destination
+system. To be able to transfer the checkpoint it is possible to specify an output-file.
```

Comment:
nit:  Suggest s/To be able to transfer the checkpoint it is possible to specify an output-file./When transferring the checkpoint, it is possible to specify an output-file./

Reply:
changed